WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift

```diff
@@ -151,11 +147,12 @@ struct WooShippingCustomsForm: View {
                     Button {
                         // TODO: Save values
                         presentationMode.wrappedValue.dismiss()
+                        viewModel.onDismiss()
                     } label: {
-                        Text(viewModel.informationIsMissing ? Localization.addMissingInformationButtonTitle : Localization.saveCustomsDetailsButtonTitle)
+                        Text(viewModel.requiredInformationIsEntered ? Localization.saveCustomsDetailsButtonTitle : Localization.addMissingInformationButtonTitle)
```

Comment:
I changed the property from `informationIsMissing` to `requiredInformationIsEntered` which is easier to use.